src/Network/Network/help/New-AzApplicationGatewayRewriteRuleUrlConfiguration.md

```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/new-azapplicationgatewayrewriteruleurlconfiguration
+schema: 2.0.0
+---
+
+# New-AzApplicationGatewayRewriteRuleUrlConfiguration
+
+## SYNOPSIS
+Creates a rewrite rule url configuration for an application gateway.
+
+## SYNTAX
+
+```
+New-AzApplicationGatewayRewriteRuleUrlConfiguration [-ModifiedPath <String>] [-ModifiedQueryString <String>] [-Reroute]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+**The AzApplicationGatewayRewriteRuleUrlConfiguration** cmdlet creates a rewrite rule url configuration for an Azure application gateway.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $urlCongiguration = New-AzApplicationGatewayRewriteRuleUrlConfiguration -ModifiedPath /abc -ModifiedQueryString x=y&a=b
```

Comment:
I think the ampersand could cause an error on powershell. Adding quotes around the whole parameter would work.